xapian-core/queryparser/termgenerator_internal.h

```diff
@@ -37,14 +37,15 @@ class TermGenerator::Internal : public Xapian::Internal::intrusive_base {
     Stem stemmer;
     stem_strategy strategy;
     Xapian::Internal::opt_intrusive_ptr<const Stopper> stopper;
+    stop_strategy stop_mode;
     Document doc;
     termcount termpos;
     TermGenerator::flags flags;
     unsigned max_word_length;
     WritableDatabase db;
 
   public:
-    Internal() : strategy(STEM_SOME), stopper(NULL), termpos(0),
+    Internal() : strategy(STEM_SOME), stop_mode(STOP_STEMMED), stopper(NULL), termpos(0),
```

Comment:
Building this with clang and default warnings (and `-Werror`) fails because `stop_mode` and `stopper` are the other way round in the class definition. Does this build okay on gcc?